pkgs/development/python-modules/sqlite-utils/default.nix

```
@@ -14,17 +14,17 @@
   testers,
   sqlite-utils,
 }:
-
 buildPythonPackage rec {
   pname = "sqlite-utils";
-  version = "3.36";
+  version = "3.37";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `build-system = [ setuptools ]`